docs/Ariba-batchsource.md

```diff
@@ -23,8 +23,10 @@ annotating metadata, etc.
 **Realm (M)**: Realm name from which the data is to be extracted, for example, S4VALL.   
 **View Template Name (M)**: Name of the view template from which data is to be extracted, for example, RequestFactSystemView.
 
-## Credentials:
-
+## Connection:
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+**Connection:** Name of the connection to use. Template Names information will be provided by the connection.
```

Comment:
When I view the file, these properties do not appear on separate lines. Maybe there's a missing paragraph tag or something? Connection and Ariba Client ID (M) appear in the same paragraph as the Use Connection property.

Reply:
Done the changes